src/codeGenerators/circuit/zokrates/toCircuit.ts

```diff
@@ -26,13 +26,38 @@ function codeGenerator(node: any) {
       return `${CircuitBP.uniqueify(node.imports.flatMap(codeGenerator)).join('\n')}`;
 
     case 'FunctionDefinition': {
-      const functionSignature = `def main(\\\n\t${codeGenerator(node.parameters)}\\\n) -> ():`;
+      let functionSignature : any
       const body = codeGenerator(node.body);
+      let returnStatement = `return`;
+      let returnName : string;
+      if(node.returnParameters.parameters) {
```

Comment:
```suggestion
      if(node.returnParameters) {
```
This is causing an error for me if `returnParameters` doesn't exist, just removing the `.parameters` works for me :)